src/lua/fiber.c

```diff
@@ -871,6 +871,45 @@ lbox_fiber_stall(struct lua_State *L)
 	return 0;
 }
 
+static int
+lbox_increase_deadline_timeout(struct lua_State *L)
+{
+	const uint64_t time_add = lua_tonumber(L, -1) * clock_second;
```

Comment:
Shouldn't you check arguments here, like you do in `lbox_set_default_deadline_timeout`?

Reply:
fixed